Update: Add has-numbered-pins class when using numbered pins (fix #320)

swashbuck opened the pull request:
Fixes #320 

### Fix
* Fix typo in PR template
* Add line breaks to readme

### Update
* Adds `is-numbered-pin` class when using numbered pins

### Testing
1. Enable `_useNumberedPins` on a Hot Graphic.
2. Inspect the `.hotgraphic__pin` elements and ensure the new class is present.

joe-allen-89 reviewed (approved):
👍

kirsty-hames reviewed (commented):
Nice addition thanks @swashbuck.

Should `.has-numbered-pins` be exclusive to the standard configuration only (clickable 'pin' icons overlaying the main image)? The other two layouts, [custom pin image](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/d273e8760670373f7ca2ba84a2704678305d24a2/example.json#L149) and `_useGraphicsAsPins` don't display number pins and already have their own set styles. Potentially there could be a conflict of styles but seeing as we're not attaching any styles to `.has-numbered-pins` then happy to leave as is for developers to address in their own custom styles.

cahirodoherty-learningpool reviewed (approved):
👀

swashbuck commented:
> Should .has-numbered-pins be exclusive to the standard configuration only (clickable 'pin' icons overlaying the main image)? 

@kirsty-hames That makes sense. I've moved the class to the pin buttons themselves and changed the name to `is-numbered-pin`.
